### DISCUSSION TITLE
Structure for DSMflow data exports with two calsim run

### DISCUSSION THREAD

**Discussion by ergz** [CVPIA-OSC/DSMflow#17] [General]
was thinking of the best way to transition to the new flow stuff..... what do you think about exporting the latest data using the same names and same structure so for example `DSMflow::flow_cfs` is still just a dataframe but with the latest values of flow, the DSMflow package however can also export a "hidden" dataset starting with maybe `.` so `DSMflow::..flow_cfs` will have all the data as elements of a list. I am updating the fall run model, and there is a lot breaking due to this change. Moving forward with this new structure will also require that we make sure users are using the correct version of flow package with the model otherwise it wont run. We can build functions as part of the flow or fall run package that will allow users to create a params object using either the 2008 biop or 2019 biop.  Thoughts?

**Comment by liz-fw** [comment]
So if I am understanding this correctly, the `DSMflow::..flow_cfs` would be a list with 2008 and 2019 biop datasets, and `DSMflow::flow_cfs` is just the 2019 biop data (and this applies for all the data objects in the package). We also have to modify the flow/fall run package functions so that users can create params from the `DSMflow::..flow_cfs` object, which contains both the original (2008) and new (2019) biop runs?
